website/docs/reference/integrations/datadog.md

```diff
@@ -28,19 +28,57 @@ You can choose to trigger updates for the following events:
 - feature-stale-off
 - feature-environment-enabled
 - feature-environment-disabled
+- feature-variants-updated
+- feature-potentially-stale-on
 
 > *) Deprecated, and will not be used after transition to environments in Unleash v4.3
 
 #### Parameters {#parameters}
 
 Unleash Datadog integration takes the following parameters.
 
-- **Datadog Events URL** - This property is optional. The default url is https://api.datadoghq.com/api/v1/events. Needs to be changed if you are not not on the US1 [Datadog site](https://docs.datadoghq.com/getting_started/site/). Possible alternatives:
+- **Datadog Events URL** - This property is optional. The default URL is https://api.datadoghq.com/api/v1/events. Needs to be changed if you are not not on the US1 [Datadog site](https://docs.datadoghq.com/getting_started/site/). Possible alternatives:
   - EU: https://app.datadoghq.eu/api/v1/events
   - US1: https://app.datadoghq.com/api/v1/events
   - US3: https://us3.datadoghq.com/api/v1/events
   - US1-FED: https://app.ddog-gov.com/api/v1/events
-- **DD API KEY** - This is a required property.
+
+
+- **Datadog API key** - This is a required property. The API key to use to authenticate with Datadog.
+
+- **Datadog Source Type Name** - This property is optional. Sets `source_type_name` parameter to be included in Datadog events.
+
+- **Extra HTTP Headers** - This property is optional. Used to set the additional headers when Unleash communicates with Datadog.
+
+Example:
+
+```json
+{
+  "ISTIO_USER_KEY": "hunter2",
+  "SOME_OTHER_CUSTOM_HTTP_HEADER": "SOMEVALUE",
+}
+```
+
+- **Body template** - This property will be introduced as `beta` in `5.5` and is expected to enter `GA` in `5.6`. It is optional. Used to override the body template used by Unleash when performing the HTTP POST. You may format you message using a [Mustache template](https://mustache.github.io). You will have the [Unleash event format](/reference/api/legacy/unleash/admin/events) available in the rendering context.
```

Comment:
Suggestion: combination of moving availability and some slight rewording.

But what is the "unleash event format"? What are you linking to that page of the docs for? If you're looking for information on event shapes, they're on [`/reference/event-types`](https://docs.getunleash.io/reference/event-types) instead. I don't think I quite understand what you're saying. Is it that you'll have the event's properties available? In that case, maybe something like this?

```suggestion
- **Body template** - This is in optional property. The template is used to override the body template used by Unleash when performing the HTTP POST. You can format your message using a [Mustache template](https://mustache.github.io). Refer to the [Unleash event types](/reference/event-types) reference to find out which event properties you have access to in the template.
- ```

Reply:
Agreed. I think this was simply a copy-paste from `webhook.md`, so maybe we should update that one as well at some point. Addressed in https://github.com/Unleash/unleash/pull/4802/commits/a77a3698518082dc51032194bc28b89d4b60e5dc